app/params/encoding.go

```diff
@@ -10,7 +10,7 @@ import (
 // This is provided for compatibility between protobuf and amino implementations.
 type EncodingConfig struct {
 	InterfaceRegistry types.InterfaceRegistry
-	Marshaler         codec.Codec
+	Codec             codec.Codec
```

Comment:
Is this just a renaming?

Reply:
Yes, it slipped into this PR and with all the rebasing was to difficult to separate out again, sorry